samples/mcuboot/firmware_loader_entrance/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+  name: Firmware loader entrance
+  description: Demo application for entering the firmware loader application
+common:
+  sysbuild: true
+  build_only: true
+tests:
+  sample.mcuboot.firmware_loader_entrance:
+    harness: bluetooth
```

Comment:
there is no bleutooth harness in twister, so twister will skip that tests. Please use harness: console, to check at least if sample is up and running on device